test/e2e/proxy_test.go

```diff
@@ -744,6 +744,7 @@ func testProxyHairpin(t *testing.T, isIPv6 bool) {
 	agnhostHost := fmt.Sprintf("agnhost-host-%v", isIPv6)
 	createAgnhostPod(t, data, agnhostHost, node, true)
 	t.Run("HostNetwork Endpoints", func(t *testing.T) {
+		skipIfKubeProxyEnabled(t, data)
```

Comment:
why this is needed?

Reply:
Previously, this test is ran when kube-proxy is removed and Antrea Proxy handles ClusterIP traffic. Now, as the kube-proxy presents and handles ClusterIP traffic, the result of the test won't be the expected result.